recipes/VideoGenPage.py

```
@@ -1,27 +1,38 @@
 from __future__ import annotations
 
+import json
 import json
```

Ruff (0.14.1)

Redefinition of unused `json` from line 3

Remove definition: `json`

(F811)